convert2rhel_insights_tasks/main.py

```diff
@@ -674,7 +674,13 @@ def prepare_environment_variables(env):
     :type env: dict[str, Any]
     """
     for variable, value in env.items():
-        if variable.startswith("CONVERT2RHEL_") and value == 0:
+        # We can pop out of context both the OPTIONAL_REPOSITORIES and
+        # ELS_DISABLED envs as they are not necessary for convert2rhel
+        # execution.
+        if variable in ("OPTIONAL_REPOSITORIES", "ELS_DISABLED"):
+            env.pop(variable)
```

Comment:
This condition can stay but it seems it doesn't need to be here. Leaving the vars there will cause no harm.

Reply:
I don't want those extra properties being propagated to the env context for convert2rhel. As clean as we can make it, I want it to be.

Reply:
```suggestion
        # We can pop out any env var that is not necessary for convert2rhel execution
        if not variable.startswith("CONVERT2RHEL_"):
            env.pop(variable)
```
Ack. So what about making it more generic?